Source/Particles/Collision/BackgroundMCCCollision.H

```diff
@@ -57,8 +58,10 @@ public:
 
 private:
 
-    VectorType< MCCProcess* > m_scattering_processes;
-    VectorType< MCCProcess* > m_ionization_processes;
+    amrex::Vector<std::unique_ptr<MCCProcess> > m_scattering_processes;
+    amrex::Vector<std::unique_ptr<MCCProcess> > m_ionization_processes;
```

Comment:
I think these could just be `amrex::Vector<MCCProcess>`

Reply:
Yes, good point.  I will update.  My original concern was if MCCProcess is not movable, the copy will involve gpu device copy operation.  But it is movable.